Activities/Cryptography/UiPath.Cryptography/Properties/UiPath.Cryptography.resx

```diff
@@ -144,6 +144,9 @@
   <data name="HMACSHA512" xml:space="preserve">
     <value>HMACSHA512</value>
   </data>
+  <data name="KEY" xml:space="preserve">
```

Comment:
Why the all caps resource name? Same for SecureKey.

Reply:
No reason, got tricked by algorithms names there. Changed it.